apps/core-angular-cli/tsconfig.json

```diff
@@ -6,16 +6,18 @@
     "outDir": "./dist/out-tsc",
     "forceConsistentCasingInFileNames": true,
     "strict": true,
+    "noImplicitAny": false,
```

Comment:
The changes here mirror the tsconfig file for packages/angular tsconfig. I made them similar while trying to figure out why two tests were failing.